azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -785,7 +784,7 @@ public void removePermission(final Project project, final String name, final boo
 
   @Override
   public List<Triple<String, Boolean, Permission>> getProjectPermissions(
```

Comment:
I searched and recalled that this method has to be used in my test. So I didn't remove it.